### DISCUSSION TITLE
Hi, can I set github event types ?

### DISCUSSION THREAD

**Discussion by medmin** [sarisia/actions-status-discord#356] [General]
There are lots of github events, such as push, issue,merge, etc

But maybe I am only interested in some certain events, such as new issue. 

So, is it possible for me to just set the github events and it only send messages when the selected events happen ? And it can ignore other events. 

Thanks

**Comment by sarisia** [comment]
Hi @medmin and sorry for my late reply!

If you want to run workflow for only certain events, try modifying `on` in your workflow to limit
triggers. See: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on

Else, if you want to run workflow for all events but only want to limit sending Discord messages,
try modifying `if` condition of the action step. See: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

Hope it helps!
